sources/packages/backend/apps/workers/src/controllers/program-info-request/_tests_/e2e/program-info-request.controller-updateApplicationStatus.e2e-spec.ts

```diff
@@ -77,6 +93,304 @@ describe("ProgramInfoRequestController(e2e)-updateApplicationStatus", () => {
+          programName: "valueA",
+          programDescription: "valueB",
+          programPersistentProperties: ["programName", "programDescription"],
+        } as ApplicationData,
+      },
+    );
+    const updateApplicationStatusPayload = createFakeUpdatePIRStatusPayload(
+      pirApplicationCurrent.id,
+      pirOffering.educationProgram.id,
+      ProgramInfoStatus.required,
+    );
+
+    // Act
+    const result = await programInfoRequestController.updateApplicationStatus(
+      createFakeWorkerJob<
+        ProgramInfoRequestJobInDTO,
+        ProgramInfoRequestJobHeaderDTO,
+        ProgramInfoRequestJobOutDTO
+      >(updateApplicationStatusPayload),
+    );
+
+    // Asserts
+    expect(result).toEqual({
+      resultType: MockedZeebeJobResult.Complete,
+      outputVariables: { programInfoStatus: ProgramInfoStatus.completed },
+    });
+
+    // Asserts that the application PIR status is completed
+    // and data was updated as expected.
+    const expectedApplication = await db.application.findOne({
+      select: {
+        id: true,
+        pirStatus: true,
+        pirHash: true,
+        pirProgram: { id: true },
+        pirAssessedDate: true,
+        pirAssessedBy: { id: true },
+        pirApprovalReference: { id: true },
+        currentAssessment: {
+          id: true,
+          offering: { id: true },
+          updatedAt: true,
+          modifier: { id: true },
+        },
+        updatedAt: true,
+        modifier: { id: true },
+        student: {
+          id: true,
+          notes: { id: true, description: true, noteType: true },
+        },
+      },
+      relations: {
+        pirProgram: true,
+        pirAssessedBy: true,
+        pirApprovalReference: true,
+        currentAssessment: { offering: true, modifier: true },
+        modifier: true,
+        student: { notes: true },
+      },
+      where: { id: pirApplicationCurrent.id },
+      loadEagerRelations: false,
+    });
+    expect(expectedApplication).toEqual({
+      id: pirApplicationCurrent.id,
+      pirStatus: ProgramInfoStatus.completed,
+      pirHash,
+      pirProgram: { id: pirOffering.educationProgram.id },
+      pirAssessedDate: now,
+      pirAssessedBy: systemUsersService.systemUser,
+      pirApprovalReference: { id: pirApplicationVersion.id },
+      updatedAt: now,
+      modifier: systemUsersService.systemUser,
+      currentAssessment: {
+        id: pirApplicationCurrent.currentAssessment.id,
+        offering: { id: pirOffering.id },
+        updatedAt: now,
+        modifier: systemUsersService.systemUser,
+      },
+      student: {
+        id: expectedApplication.student.id,
+        notes: [
+          {
+            id: expect.any(Number),
+            description: `The program information request was automatically completed using information from a previous request that was approved on ${getDateOnlyFormat(
+              pirAssessedDate,
+            )}.`,
+            noteType: NoteType.Application,
```

Comment:
Can we have the AC updated to specify the note type? 
<img width="823" height="109" alt="image" src="https://github.com/user-attachments/assets/0d45e873-76c7-4a3c-b516-d3024415237e" />

Reply:
Asked biz.